proposals/lip-0028.md

```diff
@@ -371,7 +377,8 @@ This proposal introduces a hard fork in the network. Since the transaction seria
 
 ```js
 myTrs = {
```

Comment:
I can only comment here. I think that the binary of `myTrs` below should also be updated.

Reply:
Was the binary updated and reviewed?

Reply:
Yes, as per the last commit.